ghost/core/core/server/services/url/Resources.js

```diff
@@ -95,28 +94,28 @@ class Resources {
      * 3 event listeners connected to "_onResourceUpdated" handler and it's 'tag.edited', 'tag.attached', 'tag.detached' events
      * 1 event listener connected to  "_onResourceRemoved" handler and it's 'tag.deleted' event
      */
-    initEvenListeners() {
+    async initEventListeners() {
```

Comment:
I think we can do without the `async` here, and the corresponding `await`s in `ghost/core/core/server/services/url/UrlService.js` - this function doesn't do anything asynchronous, or return a Promise 🙂 (an upcoming lint rule addition to Ghost would have caught this)

Reply:
Good spot fix in 5371dc4